test/cpp/lazy/test_lazy_ops.cpp

```diff
@@ -78,6 +80,18 @@ static inline at::DeviceType DefaultDevice() {
 
 }  // namespace
 
+TEST(LazyDynamicOpsTest, NarrowCopy) {
+  auto x = torch::rand({5, 10, 10}).to(kLazy);
+  const size_t Y_DIM = 3;
+  const size_t X_DIM_INDEX = 2;
+  auto y = torch::rand({Y_DIM}).to(kLazy);
+  auto ly = torch::lazy::TryGetLtcTensor(y);
+  auto dim_node = MakeNode<SizeNode>(ly->GetIrValue(), 0);
+  auto lmn = std::make_shared<torch::lazy::SymbolicIntNode>(dim_node);
+  auto z = x.narrow_copy(X_DIM_INDEX, 0, lmn->toSymInt());
```

Comment:
Trying to locate how exactly `lmn->toSymInt()`is propagated to the `Lower` function. Would be great to discuss it in our meeting tomorrow.

Reply:
Assuming the `SymInt` parameter allows a call to the upstream `SizeNode` object via the `toSymbolicIntNode()`  API.

Reply:
yeah this is exactly how it works! Hopefully, it's intuitive enough API, but we can work on it.